scripts/release/Jenkinsfile

```diff
@@ -0,0 +1,99 @@
+pipeline {
+    environment {
+        AWS_ACCESS_KEY_ID = credentials("aws-access-key-id")
+        AWS_SECRET_ACCESS_KEY = credentials("aws-secret-access-key")
+    }
+
+    agent any
+
+    stages {
+        stage("create ec2 instance") {
+            steps {
+                sh script: 'scripts/release/start_ec2_instance.sh us-west-1 ami-0dd655843c87b6930 t2.2xlarge'
+            }
+        }
+
+        stage("setup ec2 instance") {
+            steps {
+                sh 'aws s3 cp s3://algorand-devops-misc/tools/gnupg2.2.9_centos7_amd64.tar.bz2 .'
```

Comment:
it's ok for now, but I would prefer to move this away from S3 and into github.
( like the reminder of our build sources ).

Reply:
Ok, I'll either move this iteration or make a note to do so.

Reply:
I built that compilation of gpg for centos. Centos7 ships with a version that is too old. I statically compiled a gpg that we can unpack into Centos7 and run. S3 is the right place for that archive.